common/rpc/test/rpc_localstore_tls_test.go

```diff
@@ -689,3 +733,46 @@ func (s *localStoreRPCSuite) validateTLSInfo(tlsInfo *credentials.TLSInfo, err e
 	sn := (*tlsInfo.State.PeerCertificates[0].SerialNumber).Int64()
 	s.Equal(serialNumber, sn)
 }
+
+func (s *localStoreRPCSuite) TestRingpopMutualTLS() {
+	runRingpopTLSTest(s.Suite, s.logger, s.ringpopMutualTLSRPCFactoryA, s.ringpopMutualTLSRPCFactoryB, true)
+}
+
+func (s *localStoreRPCSuite) TestRingpopServerTLS() {
+	runRingpopTLSTest(s.Suite, s.logger, s.ringpopServerTLSRPCFactoryA, s.ringpopServerTLSRPCFactoryB, true)
+}
+
+func (s *localStoreRPCSuite) TestRingpopInvalidTLS() {
+	runRingpopTLSTest(s.Suite, s.logger, s.insecureRPCFactory, s.ringpopServerTLSRPCFactoryB, false)
+}
+
+func runRingpopTLSTest(s suite.Suite, logger log.Logger, serverA *TestFactory, serverB *TestFactory, isValid bool) {
```

Comment:
I think it would be better if:
```suggestion
func runRingpopTLSTest(s suite.Suite, logger log.Logger, serverA *TestFactory, serverB *TestFactory, expectError bool) {
```
Also flip values on caller side and `if` checks bellow.